pkg/deploy/apis/apps/types.go

```diff
@@ -123,6 +123,7 @@ const (
 )
 
 // +genclient
+// +genclient:createInstantiateVerb=subresource=instantiate,input=DeploymentRequest
```

Comment:
This only works if DeploymentRequest is in the same package (does not work for external types or kapi / etc.) ...

Reply:
> This only works if DeploymentRequest is in the same package (does not work for external types or kapi / etc.)

One our our use-cases is Scale (in autoscaling) and another is ImageSecrets or some such (in the core API group).  Could we provide a full package name or something?